website/public/js/controllers/groupsCtrl.js

```diff
@@ -344,7 +344,21 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
           });
         });
       }
-    }
+    };
+
+    $scope.copyToDo = function(message) {
+      var taskNotes = '[' + message.user + '](' + env.BASE_URL + '/static/front/#?memberId=' + message.uuid +
```

Comment:
Since these get put in notes, and the notes popover isn't persistent once you move off of a task, it's impossible to actually click on the link. :(

Should we make the popover persistent when the notes icon is clicked?

Zapping a chat from the Tavern says it came from HabitRPG. Should we have a special rule around that to localize Tavern for the user's particular language?

Should we also have a translatable string for `foo wrote in bar`?